tests/config-array.test.js

```diff
@@ -690,6 +689,35 @@ describe('ConfigArray', () => {
 				expect(config).to.be.undefined;
 			});
 
+			// https://github.com/eslint/eslint/issues/17103
+			it('should return config when a filename matches an ignores pattern only', () => {
```

Comment:
Same as the previous comment, I don't think this test should pass. In https://github.com/eslint/eslint/issues/17103, the problem is that config with `ignores: ["error.js"]` applies to `error.js`, not that it doesn't add `error.js` as a lint target when searched with globs.